app/state_machines/flood_risk_engine/work_flow.rb

```diff
@@ -0,0 +1,71 @@
+module FloodRiskEngine
+  # Work flows are sequences of steps
+  # This class is designed to allow work flows to be defined as arrays,
+  # and then be converted into a hash suitable for input into a state
+  # engine.
+  class WorkFlow
+    # Define each work flow within Definitions
+    module Definitions
+      extend self
+
+      def local_authority
+        start + local_authority_branch + finish
+      end
+
+      def start
+        [
+          :check_location,
+          :check_exceptions,
+          :add_exceptions,
+          :grid_reference,
+          :user_type
+        ]
```

Comment:
I used organisation_type instead if user_type - there is a working organisation_type_form ready but I could be renamed to user_type_form. We should probably have a go at nailing down what we want to call the steps (independent of the prototype) sooner rather than later so the step names don't confuse other developers down the line.